api.md

```diff
@@ -47,7 +47,7 @@ window will be closed automatically.</p>
 <a name="start"></a>
 
 ## start(options) ⇒ <code>Promise&lt;Object&gt;</code>
-Start the hyper sdk with configurations. It will init the hyper sdk, and login user with the auth options, then enable cards specified in the configurations, if not specified, it will enable all the certified cards or owned cards by default.
+Start the hyper sdk with configurations. It will init the hyper sdk, and login user with the auth options, then enable cards specified in the configurations, if not specified, it will enable all the certified cards or owned cards by default. After cards get enabled, it will regularly check if card data needs update and do the refresh if needed, if not specified, it will always enable the auto fresh, do the check every 5mins.
 
```

Comment:
Let's change this to:
"The Hyper SDK is designed to update card data without manual intervention seamlessly. It initiates an automatic refresh every 5 minutes, examining data and card definitions for any modifications. Upon detecting a change, it triggers an automatic refresh to incorporate the updated information. Users can customize this automatic refresh behavior through the available "refresh" options."